one-ui/src/main/resources/application.properties

```diff
@@ -12,3 +12,7 @@ server.servlet.session.timeout=5m
 server.servlet.session.cookie.max-age=-1
 
 server.servlet.session.cookie.name=DHOneUISession
+
+spring.servlet.multipart.max-file-size=100MB
+
+spring.servlet.multipart.max-request-size=100MB
```

Comment:
Might want to raise the max request slightly above 100MB to handle multiple files that add up to 100mb themselves, but fall a few bytes over the request limit.

Also, we may want to look at a better way than just to 'hang' the app while uploading a huge file. Most systems use a form of streaming service in the background, not rest, to gracefully handle a fileupload in the background.

Reply:
@aebadirad - I can raise it to 105 MB approx. That should avoid such scenarios where the overall size of multiple files is slightly above the 100 MB limit.